Add ability to put previews into sidecar directory (#2061)
ViewComponent/view_component: docs/CHANGELOG.md

## main

## 3.13.1

> * Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory.

Reviewer: ```suggestion
* Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.
```

> * Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.

Reviewer: ```suggestion
* Look for preview files that end in `preview.rb` rather than `_preview.rb` to allow previews to exist in sidecar directory with test files.
```

> ## 3.13.1
> 
> * Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.
> 

Reviewer: ```suggestion
* Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.

    *Seth Herr*

```
